.github/workflows/rust.yml

```diff
@@ -116,7 +116,7 @@ jobs:
         run: cargo test -p alvr_session
 
   check-format:
-    runs-on: ubuntu-24.04
```

Comment:
No, there is nothing actually being pulled in from the system, this should stay 24